shub/apps/main/views/download.py

```diff
@@ -116,8 +112,28 @@ def download_container(request,cid,secret):
     if container.secret != secret:
         return Http404
 
+    return _download_container(container)
+
+
+def _download_container(container):
```

Comment:
@dctrud see here, since we are doing the same thing in two functions, I  thought this was a good use for a shared ("private" to download.py) function. I also added the content length so that the progress bar comes back, eg here it is with it:

```
sregistry pull peanut-butter/jelly:time
[client|registry] [database|sqlite:////home/vanessa/.singularity/sregistry.db]
Progress |===================================| 100.0% 
[container][update] peanut-butter/jelly:time
Success! /home/vanessa/.singularity/shub/peanut-butter-jelly:time.simg
```

and without is just empty space

```
sregistry pull peanut-butter/jelly:time
[client|registry] [database|sqlite:////home/vanessa/.singularity/sregistry.db]

[container][update] peanut-butter/jelly:time
Success! /home/vanessa/.singularity/shub/peanut-butter-jelly:time.simg
```

Reply:
Yup - LGTM.